src/fsharp/service/service.fs

```diff
@@ -524,7 +524,15 @@ type TypeCheckInfo
             let item = getItem x
             let n1 =  item.DisplayName 
             match item with
-            | Item.Types _ | Item.CtorGroup _ -> residue + "Attribute" = n1 || residue = n1
+            | Item.Types _ -> residue + "Attribute" = n1 || residue = n1
```

Comment:
Not sure how often this is called but can you please change the order so that string addition is only done on second position?

Reply:
Done.